### PR TITLE
Moves texts from html to _config.yml

### PR DESCRIPTION
**Pull request by erickmendonca**
Moves most texts from HTML to _config.yml. It makes easier to create site on other languages.

Minor updates: removes server and auto options, changes highlighter.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -24,9 +22,13 @@ site:
     cover: "http://f.cl.ly/items/2X28422q1e3w0C2U1P3H/866591_24254643.jpg"
     facebook: "http://braziljs.org/img/fb-share.jpg"
 
+subscribeText: "Register now"
```
Comment (maurogeorge) [inline _config.yml]: Could you please update the `subscribeText` to `subscribe_text`? And all other variables definition that follow this convention? It is a good practice define variables and symbols in ruby using [snake_case](https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)

File: _config.yml [hunk 2]
```diff
@@ -118,4 +120,65 @@ partners:
...
+locationSection:
+   title: "Location"
+
+# Partners section
+partnersSection:
+   title: "Partners"
+   showYourLogoHere: true
+
+# Schedule section
+scheduleSection:
```
Comment (maurogeorge) [inline _config.yml]: You missed the call of this definitions on the [_includes/section/schedule.html](https://github.com/maurogeorge/conf_boilerplate_ruby/blob/master/_includes/section/schedule.html)

Comment (maurogeorge) [inline _config.yml]: Could you please fix this =D

File: _includes/nav.html [hunk 3]
```diff
@@ -3,37 +3,42 @@
   <ul class="wrapper">
     {% if site.sections.about %}
       <li class="nav-item">
-        <a href="#about" class="nav-link">About</a>
+        <a href="./#about" class="nav-link">{{site.aboutSection.title}}</a>
```
Comment (maurogeorge) [inline _includes/nav.html]: I guess we dont need this `./` before the anchors? Could you explain why you add this?

File: _includes/section/about.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
   <!-- ABOUT -->
   <section class="about" id="about">
-    <h2 class="section-title">About</h2>
+    <h2 class="section-title">{{site.aboutSection.title}}</h2>
     <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
```
Comment (maurogeorge) [inline _includes/section/about.html]: I guess we can add this paragraphs to the `_config` too.

**Comment by maurogeorge** [comment]
@erickmendonca thanks for your job :star2:
I did some review, could you please update the code?

**Comment by erickmendonca** [comment]
Great! I'll do soon.

Em seg, 19 de jan de 2015 18:47, Mauro George <notifications@github.com>
escreveu:

File: _config.yml (hunk 1 above)
Comment (erickmendonca) [inline _config.yml]: It's easy to forget that!

File: _includes/nav.html (hunk 3 above)
Comment (erickmendonca) [inline _includes/nav.html]: I used this because I created new pages and that did break the navigation.

**Comment by erickmendonca** [comment]
@maurogeorge :+1: Done!

**Comment by maurogeorge** [comment]
@erickmendonca thanks a lot :star:

**Pull request merged**